src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,33 @@
+@mixin on-small {
+  @media (min-width: $small-min-width) {
+    @content;
+  }
+}
+
+@mixin on-medium {
+  @media (min-width: $medium-min-width) {
+    @content;
+  }
+}
+
+@mixin on-large {
+  @media (min-width: $large-min-width) {
+    @content;
+  }
+}
+
+@mixin content-grid-template-columns() {
+  grid-template-columns: repeat(1, 200px);
+
+  @include on-small {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-medium {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-large {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
The hard-coded value of '200px' for grid-template-columns is repeated multiple times. Consider using a variable for the column width to make the code more maintainable and to adhere to the DRY principle. For instance, you could define a variable $column-width: 200px; at the top of your file and then use it within your mixin. This way, if the column width needs to change in the future, you only have to update it in one place.